src/details/ArborX_DetailsSortUtils.hpp

```diff
@@ -52,6 +54,11 @@
 #endif   // #if defined(KOKKOS_ENABLE_CUDA)
 // clang-format on
 
+#if defined(KOKKOS_ENABLE_HIP) && defined(ARBORX_ENABLE_ROCTHRUST)
```

Comment:
Could be simplified to `#if defined(ARBORX_ENABLE_ROCTHRUST)`, as it is a dependent option.

Reply:
True but does not hurt.  Either.  Do you want me to change?

Reply:
No.